cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,399 @@
+        ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0 or len(json_out) == 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create {}' command failed\n".format(self.run_as_user) +
+                "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
+        return str(json_out[0]['key'])
+
+    def _gen_output_map(self):
+        if self.out_map:
+            return
+        self.get_cephfs_data_pool_details()
+        self.out_map['NAMESPACE'] = self._arg_parser.namespace
+        self.out_map['CLUSTER_NAME'] = self._arg_parser.cluster_name
+        self.out_map['ROOK_EXTERNAL_FSID'] = self.get_fsid()
+        self.out_map['ROOK_EXTERNAL_USERNAME'] = self.run_as_user
+        self.out_map['ROOK_EXTERNAL_CEPH_MON_DATA'] = self.get_ceph_external_mon_data()
+        self.out_map['ROOK_EXTERNAL_USER_SECRET'] = self.create_checkerKey()
+        self.out_map['CSI_RBD_NODE_SECRET_SECRET'] = self.create_cephCSIKeyring_RBDNode()
+        self.out_map['CSI_RBD_PROVISIONER_SECRET'] = self.create_cephCSIKeyring_RBDProvisioner()
+        self.out_map['CSI_CEPHFS_NODE_SECRET'] = self.create_cephCSIKeyring_cephFSNode()
+        self.out_map['CSI_CEPHFS_PROVISIONER_SECRET'] = self.create_cephCSIKeyring_cephFSProvisioner()
+        self.out_map['RGW_GATEWAY_ENDPOINT'] = self._arg_parser.rgw_endpoint
+        self.out_map['CEPHFS_POOL_NAME'] = self._arg_parser.cephfs_data_pool_name
+        self.out_map['CEPHFS_FS_NAME'] = self._arg_parser.cephfs_filesystem_name
+        self.out_map['RBD_POOL_NAME'] = self._arg_parser.rbd_data_pool_name
+        self.out_map['RGW_POOL_PREFIX'] = self._arg_parser.rgw_pool_prefix
+        self.out_map['RGW_REALM_NAME'] = self._arg_parser.rgw_realm_name
+        self.out_map['RGW_ZONEGROUP_NAME'] = self._arg_parser.rgw_zonegroup_name
+        self.out_map['RGW_ZONE_NAME'] = self._arg_parser.rgw_zone_name
+
+    def gen_shell_out(self):
+        self._gen_output_map()
+        shOutIO = StringIO()
+        for k, v in self.out_map.items():
+            if v:
+                shOutIO.write('export {}={}{}'.format(k, v, LINESEP))
+        shOut = shOutIO.getvalue()
+        shOutIO.close()
+        return shOut
+
+    def gen_json_out(self):
+        self._gen_output_map()
+        json_out = [
+            {
+                "name": "rook-ceph-mon-endpoints",
+                "kind": "ConfigMap",
+                "data": {
+                    "data": self.out_map['ROOK_EXTERNAL_CEPH_MON_DATA'],
+                    "maxMonId": "0",
+                    "mapping": {}
+                }
+            },
+            {
+                "name": "rook-ceph-mon",
+                "kind": "Secret",
+                "data": {
+                    "admin-secret": 'admin-secret',
```

Comment:
```suggestion
                    "admin-secret": "admin-secret",
```

single quote are not valid for son use double quotes instead.

Reply:
Changed...